building/export_plugin_from_eclipse.adoc

```diff
@@ -0,0 +1,37 @@
+= How to Export a plugin from Eclipse
+
+Exporting a plugin from inside eclipse is the 'quick and dirty' method to get an output binary for quick local tests. It does not generate an update site, and so this mechanism is only really suitable for local smoke-testing. 
```

Comment:
> may I suggest we just drop that and say to use the quick and correct mvn verify and pick it up from site/target 

Unfortunately, this method isn't "quick" in the way I'm thinking of it. It involves switching to command line, running a full build for all plugins (rather than just the one you want), using the menus to install from an update site, browsing to it, then restarting.  This requires switches to command line, then switches back to eclipse, then browsing hard drive folders, etc.  The quick-and-dirty is really quick, and kinda dirty, but it's very nice for zero-turnaround situations and saves me a lot of time while debugging hard problems.